backend/setup.go

```diff
@@ -136,30 +156,87 @@ func SetupTracer(pluginID string, tracingOpts tracing.Opts) error {
 		// Initialize global tracer for plugin developer usage
 		tracing.InitDefaultTracer(otel.Tracer(pluginID))
 	}
-	Logger.Debug("Tracing", "enabled", tracingCfg.IsEnabled(), "propagation", tracingCfg.Propagation)
+
+	Logger.Debug(
```

Comment:
Nit. Would it make sense to log tracing enabled with args when enabled and either skip when disabled or only log "Tracing not enabled"? Or "Tracing configured" as the message to add some more context

Reply:
Yes but I would prefer also doing it for the profiler:

https://github.com/grafana/grafana-plugin-sdk-go/blob/515cdb474049259f95d6714c24d08beb62faeac2/backend/setup.go#L65

I originally did it this way because I wanted to keep it consistent with this :point_up: log line

Reply:
Or maybe like this?

```diff
diff --git a/backend/setup.go b/backend/setup.go
index c659cce..27bee8c 100644
--- a/backend/setup.go
+++ b/backend/setup.go
@@ -157,15 +157,18 @@ func SetupTracer(pluginID string, tracingOpts tracing.Opts) error {
                tracing.InitDefaultTracer(otel.Tracer(pluginID))
        }
 
-       Logger.Debug(
-               "Tracing",
-               "enabled", tracingCfg.isEnabled(),
-               "propagation", tracingCfg.propagation,
-               "samplerType", tracingCfg.sampler.SamplerType,
-               "samplerParam", tracingCfg.sampler.Param,
-               "samplerRemoteURL", tracingCfg.sampler.Remote.URL,
-               "samplerRemoteServiceName", tracingCfg.sampler.Remote.ServiceName,
-       )
+       enabled := tracingCfg.isEnabled()
+       Logger.Debug("Tracing", "enabled", enabled)
+       if enabled {
+               Logger.Debug(
+                       "Tracing configuration",
+                       "propagation", tracingCfg.propagation,
+                       "samplerType", tracingCfg.sampler.SamplerType,
+                       "samplerParam", tracingCfg.sampler.Param,
+                       "samplerRemoteURL", tracingCfg.sampler.Remote.URL,
+                       "samplerRemoteServiceName", tracingCfg.sampler.Remote.ServiceName,
+               )
+       }
        return nil
 }
```